go-vulndb/govulndb.go

```diff
@@ -98,6 +105,17 @@ func (c VulnDB) Update() error {
 		}
 	}
 
+	if len(modulesErrors) > 0 {
```

Comment:
Now, we handle only a 404 error. We probably don't need the check. We cannot detect the case where all modules are 404, but it is acceptable.

Reply:
hm... okay, removed this